test/testModel.cc

```diff
@@ -1285,8 +1285,8 @@ SUITE(GodleyTableWindow)
       CHECK_EQUAL("r2c2",godleyIcon->table.cell(2,2));
       CHECK_EQUAL("r1c2",godleyIcon->table.cell(1,2));
       // Not sure why column 1's contents cannot be tested????
-      CHECK_EQUAL("r2c1",godleyIcon->table.cell(2,1));
-      CHECK_EQUAL("r1c1",godleyIcon->table.cell(1,1));
+      //CHECK_EQUAL("r2c1",godleyIcon->table.cell(2,1));
+      //CHECK_EQUAL("r1c1",godleyIcon->table.cell(1,1));
```

Comment:
Column 1 has been moved to column 3 at line 1274. But I think it is enough of a test to test on column 2, like you do, so this code can be removed.

Reply:
So no wonder they appear empty in the comparison, I never defined initial "r*c3" values for column 3. I might not have been surprised to see a mismatch, but was confused by the fact that the comparison was with empty values. Thanks for clearing that up. :)